aspnetcore/fundamentals/logging/index.md

```diff
@@ -287,6 +287,25 @@ public class Program
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-3.1"
+
+### Create logs in Blazor WebAssembly
+
+Configure logging in Blazor WebAssembly apps with the `WebAssemblyHostBuilder.Logging` property in `Program.Main`:
+
+```csharp
+using Microsoft.Extensions.Logging;
+
+...
+
+builder.Logging.SetMinimumLevel(LogLevel.Debug);
```

Comment:
Perhaps we should show the line `var builder = WebAssemblyHostBuilder.CreateDefault(args);` to make it clear that `builder` has to be `WebAssemblyHostBuilder` type.